src/components/DraggableElement/index.tsx

```diff
@@ -54,39 +73,50 @@ const DraggableElement: FC<Props> = ({ id }) => {
 
       dispatch(
         actions.webSocket.emit.game.positionUpdate({
-          x,
+          x: x - xOffset,
           y,
           id
         })
       );
     }
-  }, [dispatch, id]);
+  }, [dispatch, id, xOffset]);
 
   // effects
 
   useEffect(() => {
     if (!isDragging) {
       gsap.to($element.current, {
-        x: elementPosition.x,
-        y: elementPosition.y,
+        x: x + xOffset,
+        y: y,
         duration: gameTick / 1000,
         ease: Linear.easeNone
       });
     }
-  }, [elementPosition.x, elementPosition.y, gameTick, isDragging]);
+  }, [gameTick, isDragging, x, xOffset, y]);
+
+  useEffect(() => {
```

Comment:
Question con surement, mais du coup pourquoi tu l'as pas fait en css ?

Reply:
Alors en effet on j'aurais pu le faire en CSS (et c'est d'ailleurs ce que j'ai fait au tout début), mais je l'ai fait avec gsap pour plusieurs raisons :
- pour les performances
- pour la simplicité / lisibilité du code
- pour le fait qu'in fine avec PIXI on va manipuler des objects et non pas du DOM html du coup on ne pourra pas utiliser du CSS pour faire nos animations